lib/datasets/aozora-bunko.rb

```diff
@@ -0,0 +1,163 @@
+require_relative 'dataset'
+
+module Datasets
+  # Dataset for AozoraBunko
+  class AozoraBunko < Dataset
+    # 人物ID,著者名,作品ID,作品名,仮名遣い種別,翻訳者名等,入力者名,校正者名,状態,状態の開始日,底本名,出版社名,入力に使用した版,校正に使用した版
+    Record = Struct.new(
+      :novelist_id,
+      :novelist,
+      :title_id,
+      :title,
+      :syllabary_spelling_type, # 仮名遣い種別
+      :translator_name,
+      :registered_person_name,
+      :proofreader_name,
+      :status,
+      :status_specified_date,
+      :original_book_name,
+      :original_book_publisher_name,
+      :used_version_for_registration,
+      :used_version_for_proofreading
+    )
+
+    RecordExtended = Struct.new(
+      # 作品ID,作品名,作品名読み,ソート用読み,副題,副題読み,原題,初出,分類番号,文字遣い種別,作品著作権フラグ,公開日,最終更新日,図書カードURL,
+      :title_id,
+      :title,
+      :title_reading,
+      :title_reading_collation,
+      :sub_title,
+      :sub_title_reading,
+      :original_title,
+      :first_appearance,
+      :ndc_code, # 分類番号(日本十進分類法の番号)
+      :syllabary_spelling_type, # 文字遣い種別, Recordの仮名遣い種別 と同じ値になっている
+      :copyright_for_title,
+      :published_date,
+      :latest_updated_date,
+      :detail_url,
+      # 人物ID, 姓,名,姓読み,名読み,姓読みソート用,名読みソート用,姓ローマ字,名ローマ字,役割フラグ,生年月日,没年月日,人物著作権フラグ,
+      :novelist_id,
```

Comment:
How about using `person` instead of `novelist? Because file name uses `person` (`list_person_all_utf8.zip`) not `novelist`. It seems that there are some not novel titles such as https://www.aozora.gr.jp/cards/001428/files/50328_64360.html .

```suggestion
      :person_id,
```

Reply:
done.